docker/0.3.0/fedimintd-mainnet-tls/meta.json

```diff
@@ -0,0 +1,5 @@
+{
+  "<federation_id_hex>": {
+    "federation_name": "Override Federation Name"
+  }
+}
```

Comment:
TODO: generate that file automatically once the federation is set up

Reply:
Do you want to land this change before merge or add a TODO for a follow up PR? @elsirion

Reply:
After thinking about it a bit more: it's not bad enough UX to worry about given that we are phasing out meta override files anyway